app/controllers/content.js

```diff
@@ -40,6 +41,14 @@ function queryStringify(queryParams) {
     return query.join('&');
 }
 
+/**
+ * @module ember-preprints
+ * @submodule controllers
```

Comment:
I *think* declaring the module once at the top of the file is sufficient and might be preferred?